SwiftLSPClient/Types/TextSynchronization.swift

```diff
@@ -88,3 +88,18 @@ public struct DidCloseTextDocumentParams: Codable {
         self.init(textDocument: docId)
     }
 }
+
+public struct ReferencesParams: Codable {
+
+    public var textDocument: TextDocumentIdentifier
+    public var position: Position
+    public var includeDeclaration: Bool?
+
+    public init(textDocument: TextDocumentIdentifier, position: Position, includeDeclaration: Bool? = nil) {
+      self.textDocument = textDocument
+      self.position = position
+      self.includeDeclaration = includeDeclaration
+    }
+}
+
+public typealias ReferencesResponse = [Location]
```

Comment:
So, this is completely my fault. I didn't do a good job of properly grouping types initially. I started trying to fix it, but didn't make a lot of progress.

Do you think it would make more sense to add this as a new file under Types/Features?

Reply:
That's more of an editorial concern. I'd recommend you take that up. I don't really know what you mean.